bootstrap/pom.xml

```diff
@@ -54,6 +54,12 @@
         </dependency>
 
         <!-- for testing -->
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>configuration-testing</artifactId>
```

Comment:
I can't move this to the `testing` package because it's used by the `configuration` package, and it would create a circular dependency.